src/components/image/index.tsx

```diff
@@ -158,4 +167,5 @@ const styles = StyleSheet.create({
 });
 
 hoistNonReactStatic(Image, RNImage);
-export default Image;
+export {Image};
+export default asBaseComponent<ImageProps>(forwardRef(Image));
```

Comment:
Don't you want to declare that you receive the `Props` type and not only the `ImageProps`?